### PR TITLE
Add ability to count observations from file. 

### PR DESCRIPTION
**Pull request by drvinceknight**
This will duplicate some rows but that's easy to prepocess.

### CODE REVIEW

File: approximate_moran.py [hunk 1]
```diff
@@ -28,10 +35,9 @@ class ApproximateMoranProcess(axl.MoranProcess):
     Instead of playing the matches, the result is sampled
     from a dictionary of play tuples to distribution of match outcomes
     """
-    def __init__(self, players, cached_outcomes=None, turns=0,
-                 noise=0, mutation_rate=0.):
+    def __init__(self, players, cached_outcomes, mutation_rate=0.):
```
Comment (marcharper) [inline approximate_moran.py:38]: Not sure why we're fixing the parameters here -- we'll want to set `noise != 0` eventually.

**Review by marcharper — APPROVED** [review]
Just the one thing and then feel free to merge. Looks like I caused a merge conflict with a recent push -- perhaps it's related?

File: approximate_moran.py (hunk 1 above)
Comment (drvinceknight) [reply approximate_moran.py:38]: But these parameters are irrelevant right? They're only used with the real Moran process to create the corresponding matches but here we get all the match results from the cached outcomes.

**Comment by drvinceknight** [comment]
> Looks like I caused a merge conflict with a recent push

Have fixed the conflict, I suggest we merge this in and then merge in to `master` (this PR is to `approx`) and then stick on that.

**Pull request merged**